build/generate.mk

```
@@ -6,10 +6,16 @@ GO           ?= go
 
 PACKAGES ?= $(shell $(GO) list ./...)
 
+GOTOOLS += github.com/newrelic/tutone/cmd/tutone
+
 # Generate then lint fixes
-generate: generate-run lint-fix
+generate: generate-run generate-tutone lint-fix
+
+generate-tutone:
```

Not a blocker, but thoughts?

```suggestion
generate-types:
```

---

I'm thinking we will want to do more than types here pretty soon, so just referencing the tool here seems okay to me.